src/main/resources/interface-specification.yml

```diff
@@ -2335,17 +2335,17 @@ paths:
             application/problem+json:
```

Comment:
may I ask you to avoid moving code if not strictly required?
it's very difficult to understand if something has changed in the existing endpoints